.gitignore

```diff
@@ -5,4 +5,6 @@
 build/
 dist/
 *.egg-info/
-!.github/
\ No newline at end of file
+!.github/
+!.gitignore
+!.pre-commit-config.yaml
```

Comment:
I suspect that this file might break the build due to the missing newline. At least with the pre-commit rules you set up, this file will fail if ever updated later.

Reply:
fixed